### PR TITLE
DUPLO-16079 Bug fix

### PR DESCRIPTION
**Pull request by Duplo-nikv**
## Overview

Bug Fix
## Summary of changes
Fixed Vm not getting created
This PR does the following:

- Setting initial node count to 1 if not set at code level
- ...

## Testing performed

- [ ] Using unit tests
- [ ] Manually, on my local system
- [✓ ] Manually, on a remote test system

## Describe any breaking changes

- ...

### CODE REVIEW

**Comment by zafarabbas** [comment]
Task linked: [DUPLO-16079 TF:GCP VM/ nodes are not getting created for availability zones when autoscaling is not enabled(‘is_autoscaling_enabled = false’).](https://app.clickup.com/t/8655600/DUPLO-16079)

File: duplocloud/resource_duplo_gcp_k8_node_pool.go [hunk 1]
```diff
@@ -484,6 +484,14 @@ func expandGCPNodePool(d *schema.ResourceData) (*duplosdk.DuploGCPK8NodePool, er
 	return rq, nil
 }
 
+func setInitialNodeCount(d *schema.ResourceData) int {
```
Comment (tahir-duplo) [inline duplocloud/resource_duplo_gcp_k8_node_pool.go:487]: Can we add a default 1 at schema level instead of this logic.

Comment (Duplo-nikv) [reply duplocloud/resource_duplo_gcp_k8_node_pool.go]: default set at schema level

**Review by tahir-duplo — APPROVED** [review]
LGTM

**Pull request merged**